openslides_backend/action/actions/meeting/replace_projector_id.py

```diff
@@ -40,8 +40,11 @@ def get_updated_instances(self, payload: ActionData) -> ActionData:
             )
             changed = False
             for field in fields:
-                if meeting.get(field) == projector_id:
-                    instance[field] = meeting["reference_projector_id"]
+                change_list = meeting.get(field)
+                if change_list and projector_id in change_list:
+                    change_list.remove(projector_id)
+                    change_list.append(meeting["reference_projector_id"])
```

Comment:
only append the reference projector if the list is empty

Reply:
Done.